onnxscript/rewriter/pattern.py

```diff
@@ -193,13 +293,17 @@ def matches(self, opset):
 
     def to_ir(self, model, bindings=None) -> str:
         domain = self.domain_pattern.to_ir(model, bindings)
+        assert isinstance(domain, str), f"Expected str, got {type(domain)}"
         # TODO: Should we ban other custom domains?
-        if domain not in model.version_map:
-            model.version_map[self.domain_pattern.value] = self.version_pattern.value
+        if domain not in model.opset_imports:
+            assert not isinstance(
```

Comment:
Wonder if it is better to assert what instance it is expected to be?

Reply:
@titaiwangms

Reply:
Done